logger.go

```diff
@@ -0,0 +1,65 @@
+package traceid
+
+import (
+	"context"
+	"log/slog"
+)
+
+/*
+You can pass slog.Handler and logKey, and all the logs would automatically log
+the traceId if logging with context is used.
+
+	slog.Log()
+	slog.DebugContext()
+	slog.InfoContext()
+	slog.WarnContext()
+	slog.ErrorContext()
+
+Example:
+
+	ctx := traceid.NewContext(context.Background())
+	handler := traceid.NewLoggerWrapper(slog.NewJSONHandler(os.Stdout, nil), "traceId")
+	logger := slog.New(handler)
+	logger.InfoContext(ctx, "message")
+
+This would log
+
+	{"time":"2025-04-01T13:17:43.097789397Z","level":"INFO","msg":"message","traceId":"0195f180-2939-7bc4-bffe-838eb3c62526"}
+*/
+func NewLoggerWrapper(handler slog.Handler, logKey string) slog.Handler {
```

Comment:
I don't expect people changing the logKey very often, just like the header name, so I'd suggest to define it as
```go
var LogKey = "traceId"
```
similar to https://github.com/go-chi/traceid/blob/master/traceid.go#L10C39-L10C46

Also, let's call this `traceid.NewHandler()` or `traceid.LogHandler()`
```suggestion
func LogHandler(handler slog.Handler) slog.Handler {
```

Reply:
Good idea! Pushed.